recipes/easymt/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "easymt" %}
+{% set version = "1.0.0" %}
+package:
+  name: {{name}}
+  version: {{version}}
+
+source:
+  url: https://github.com/tang-shuyuan/EeayMT/releases/download/v{{version}}/{{name}}_v{{version}}.tar.gz
+  sha256: d9220c8399eaba7afeae9de7853f3c7a59db47c19669a9101f4b5016750b3d98
+
+build: 
+  number: 0
+  entry_points: 
+    - easymt = extract:main
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('easymt', max_pin="x") }}
+
+requirements: 
+  bulid: 
+    - python >=3.9
+  run: 
+    - blast =2.15.0
+    - flye >=2.9.4
+test: 
+  source_files: 
+    - extract.py
+    - version.py
+    - Arabidopsis_protein.fasta
+    - Drosophila_gunungcola_CM045947.fasta
+  commands:
+    - easymt -h
+
+about: 
+  home: https://github.com/tang-shuyuan/EeayMT
+  license: MIT
+  summary: "A tool for assembling mitochondrial genome"
+  description: "please contact 2421496996@qq.com if any question"
```

Comment:
This is not a good description :)

Reply:
Thanks for your reply. I have made some changes and described my software in more detail.